models/ILSVRC2012/dr_I256x256x3CMCMCMCMCMCMCMC1AS/modelargs.json

```diff
@@ -0,0 +1,14 @@
+{
+  "index": 0,
+  "num_classes": 1000,
+  "filter_size": 3,
+  "increase_factor": 112,
+  "image_size": 256,
+  "series": "d_reference",
+  "name": "dr_I256x256x3CMCMCMCMCMCMCMC1AS",
+  "model": "d_reference",
+  "trainer": "CNTK 2.2",
```

Comment:
did you switch to CNTK 2.3?  if yes may want to update the logic to add the build number.

Reply:
No I haven't made the switch yet. This model was trained with 2.2. I'll update once I switch over.